ansible/configs/sap-hana-auto/README.adoc

```diff
@@ -0,0 +1,52 @@
+== Overview
+
+*SAP HANA Auto* _config_ will cover the Technical Enablement for 'RHEL for SAP Solutions' related trainning. This is the Techincal Enablement on how to insall SAP HANA on an standalone server using both Ansible and Ansible Tower
+
+It makes an ideal base infrastructure to build on and can easily be extended via it's `env_vars.yml` to less or more machines and also to different operating system images.
```

Comment:
We now support `default_vars.yml`. Try to prefer `default_vars.yml` over `env_vars.yml` (same thing, just a different name)

if some variables are specific to a cloud provider, for example osp,  put them in `default_vars_osp.yml`.

`default_vars_ec2.yml` for aws

Reply:
We don't expect this config to be used in AWS, only OSP